File: BookStore-Angular-MongoDb/angular/src/app/book/models/book-dto.ts
import { BookType } from './book-type';import { AuditedEntityDto } from '@abp/ng.core';

export class BookDto extends AuditedEntityDto<string> {
  name: string;
  type: any;
  publishDate: string;
  price: number;
  lastModificationTime?: string;
  lastModifierId?: string;
  creationTime: string;
  creatorId?: string;
  id: string;

  constructor(initialValues: Partial<BookDto> = {}) {
    super(initialValues);
  }
}

File: DexExpress-Blazor/src/DevExpressSample.Application.Contracts/Books/BookDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace DevExpressSample.Books
{
    public class BookDto : AuditedEntityDto<Guid>
    {
        public string Name { get; set; }
        
        public DateTime PublishDate { get; set; }

        public float Price { get; set; }
    }
}